Makefile

```diff
@@ -51,15 +32,11 @@ clean:
 test:
 	go test ./... -parallel 4 
 
-e2e-test: stop-minio stop-redis run-minio run-redis
```

Comment:
nice I did this cuz of time press and wasn't able to codify the resources - happy to see it get fixed 💯